src/svgeditor/BitmapEdit.as

```diff
@@ -247,6 +247,7 @@ public class BitmapEdit extends ImageEdit {
 				// User was editing an object and switched tools, bake the object
 				bakeIntoBitmap();
 				saveToCostume();
+				targetCostume.segmentationState.unmarkedBitmap = workArea.getBitmap().bitmapData.clone();
```

Comment:
As a future enhancement, do you think we could skip this most of the time and only do it when activating the magic wand, or something like that?